_posts/en/newsletters/2020-04-15-newsletter.md

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Bitcoin Optech Newsletter #93'
+permalink: /en/newsletters/2020/04/15/
+name: 2020-04-15-newsletter
+slug: 2020-04-15-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter summarizes a proposal for creating a unified
+multi-wallet backup that circumvents the inability to import BIP32
+extended private keys into many wallets that support deterministic key
+derivation.  Also included are our regular sections describing releases
+and release candidates, changes to popular services and client software,
+and changes to popular Bitcoin infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Proposal for using one BIP32 keychain to seed multiple child keychains:**
+  several weeks ago, Ethan Kosakovsky [posted][kosakovsky post] to the
+  Bitcoin-Dev mailing list a [proposal][kosakovsky proposal] for using
+  one [BIP32][] Hierarchical Deterministic (HD) keychain to create seeds
+  for child HD keychains that can be used in different contexts.  This
+  may seem unnecessary given that BIP32 already provides extended
+  private keys (xprvs) that can be shared between signing wallets.  The
+  problem is that many wallets don't implement the ability to import
+  xprvs---they only allow importing either an HD seed or some precursor
+  data that is transformed into the seed (e.g.  [BIP39][] or [SLIP39][]
+  seed words).
+
+    Kosakovsky's proposal is to create a super-keychain whose child keys
+    are transformed into seeds, seed words, or other data that can be
+    input into various wallets' HD keychain recovery fields.  That way a
+    user with multiple wallets can backup all of them using the just the
+    super-keychain's seed (plus the derivation paths and the library for
+    transforming determinsitic entropy into input data).
+
+    {% assign img="/img/posts/2020-04-subkeychains.dot.png" %}
+    [![Using one BIP32 keychain to seed child BIP32 keychains]({{img}})]({{img}})
+
+    Reaction to the proposal was mixed, but this week one hardware wallet
```

Comment:
mixed -> perhaps describe briefly or link to reactions on the word mixed, if feasible